.github/workflows/release-container-image.yaml

```diff
@@ -0,0 +1,76 @@
+name: Release Container Image
+
+on:
+  schedule:
+    - cron: '3 3 * * *'
+  pull_request:
+  push:
+    branches: master
```

Comment:
Should be `main` though.

Reply:
This is a copy paste from back8sup  been fixed directly in main.